eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -121,8 +121,9 @@ stages:
                       - bash: |
                           npm install
                         displayName: Install versioning tool dependencies
+                        WorkingDirectory: ./eng/tools/versioning
```

Comment:
This should probably be:

```yaml
- bash: | 
    npm install
  inputs:
    workingDirectory: ./eng/tools/versioning
  displayName: Install versioning tool dependencies
```

From bash task description here: https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/bash?view=azure-devops

Reply:
actually even if i don't specify "inputs" it should be fine

Reply:
You're right! I missed the shorter syntax 🤦‍♂